src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
+    public static void printList(String[] tasks, int taskCount, int[] taskDone) {
+        String[] currList = Arrays.copyOf(tasks, taskCount + 1);
+        System.out.println("____________________________________________________________");
+        System.out.println("PENDING HIT LIST:");
+        for (int i = 0; i < currList.length; i++) {
+            if (currList[i] != null) {
+                if (taskDone[i] == 1) {
+                    System.out.println(i + 1 + "." + " [x] " + currList[i]);
+                }
+                else {
+                    System.out.println(i + 1 + "." + " [ ] " + currList[i]);
+                }
+            }
+        }
+        System.out.println("____________________________________________________________");
+
+    }
+
+    public static int doneTask(String line, String[] tasks) {
+        String[] number = line.split(" ");
+        int taskNumber = Integer.parseInt(number[1]);
+        System.out.println("TARGET NEUTRALISED: " + taskNumber + "." + " [x] " + tasks[taskNumber-1]);
+        return taskNumber-1;
+
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("____________________________________________________________");
+        System.out.println("[The Templar:]");
+        System.out.println("Greetings, Hero. Meet Vector Unit 202 - codename DUKE.\n" + "The state of the art AI assassin you requested. Give DUKE a task, and it shall be done.");
+        System.out.println("____________________________________________________________");
+
+        //declarations
+        boolean isDone = false;
+        boolean isTask = false;
+        int taskCount = 0;
+        String[] tasks = new String[100];
+        int[] taskDone = new int[100];
+
+        while (!isDone) {
+            if (isTask == true) {
+                System.out.println("[The Templar:]");
+                System.out.println("What further tasks do you require?");
+            }
+            System.out.println("____________________________________________________________");
+            System.out.println("[Hero:]");
+            String line;
+            Scanner in = new Scanner(System.in);
+            line = in.nextLine();
+
+
+            if (!line.contains("list") && !line.contains("done")) {
+                tasks[taskCount] = line;
+                taskCount++;
+            }
+
+            isTask = true; // first answer has been given
+
+            if (!line.contains("bye") && !line.contains("list") && !line.contains("done")) {
```

Comment:
Expression is a bit too long, can consider another way to handle these exceptions